python/test.py

```diff
@@ -0,0 +1,19 @@
+import ray
```

Comment:
Please remove stray file changes from the PR (here and elsewhere).

Reply:
done